addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd

```diff
@@ -207,11 +220,50 @@ func do_export_file() -> void:
 	target_file_path = gamepacks_folder + "/" + gamepack_name + ".game/default_build_menu.xml"
 	print("Exporting NetRadiant Custom Default Buld Menu to ", target_file_path)
 	file = FileAccess.open(target_file_path, FileAccess.WRITE)
+	
 	if file != null:
-		file.store_string("<?xml version=\"1.0\"?><project version=\"2.0\"></project>")
-		file.close()
-	else:
-		printerr("Error: Could not modify " + target_file_path)
+		file.store_string("<?xml version=\"1.0\"?>\n<project version=\"2.0\">\n")
+		
+		for key in default_build_variables.keys():
+			if key is String:
+				if default_build_variables[key] is String:
+					file.store_string('\t<var name="%s">%s</var>\n' % [key, default_build_variables[key]])
+				
+				else:
+					push_error("Variable key '%s' value '%s' is invalid type: %s; should be: String" % [
+						key, default_build_variables[key], 
+						type_string(typeof(default_build_variables[key]))
+						])
+			else:
+				push_error("Variable '%s' is an invalid key type: %s; should be: String" % [
+						key, type_string(typeof(key))
+						])
+			
+			
+		for key in default_build_commands.keys():
+			if key is String:
+				file.store_string('\t<build name="%s">\n' % key)
+				
+				if default_build_commands[key] is String:
+					file.store_string('\t\t<command>%s</command>\n\t</build>\n' % default_build_commands[key])
+				
+				elif default_build_commands[key] is Array:
+					for step in default_build_commands[key]:
+						if step is String:
+							file.store_string('\t\t<command>%s</command>\n' % step)
+						else:
+							push_error("Command '%s' has invalid step: %s with type: %s; should be: String" % [
+								key, step, type_string(typeof(step))
+								])	
+						
+					file.store_string('\t</build>\n')
+			
+			else:
+				push_error("Command '%s' is an invalid type: %s; should be: String" % [
```

Comment:
`Command` should be `Build option`, as reasoned above. Don't use single quotes for referencing names, use double quotes (as should be the standard throughout the rest of the plugin).